app/policies/diary_entry_policy.rb

```diff
@@ -0,0 +1,13 @@
+class DiaryEntryPolicy < ApplicationPolicy
+  def show?
+    @auth_context.role_model.is_a?(FosterParent)
+  end
+
+  class Scope < Scope
+    def resolve
+      if @auth_context.role_model.is_a?(FosterParent)
+        scope.all
```

Comment:
We'll need to only reduce the scope to the foster parent's placement diary entries.

```ruby
scope.joins(:placements).where(placements: { foster_parent_id: @auth_context.role_model.id }
```

Or something between these lines ☝️ 

+ as well, please add spec for this.

Reply:
Have had a go at it...